afs/templates/views/components/workflows/select-phys-thing-step.htm

```diff
@@ -17,15 +17,22 @@
     <!--ko if: termFilter() -->
     <div style="padding: 5px; margin-top: 15px;">
         <div class="widget-input-label">
-            <span data-bind="if not: datasetRoute">
-                Select an Object
+            <!-- ko if: datasetRoute == 'non-destructive' -->
+            <span>
```

Comment:
The combination of span and div in parallel does not seem to be necessary or less readable. Probably we can two div's.